app/models/manageiq/providers/ansible_tower/shared/automation_manager/configuration_script_source.rb

```diff
@@ -2,7 +2,14 @@ module ManageIQ::Providers::AnsibleTower::Shared::AutomationManager::Configurati
   extend ActiveSupport::Concern
 
   module ClassMethods
+    def provider_params(params)
+      authentication_id = params.delete('authentication_id') || params.delete(:authentication_id)
```

Comment:
From UI, they will be strings since they will go through http.  Keeping the symbol form here is to maintain consistency with the ATTRIBUTES we defined (they are in symbol form). 

So if backend is the caller, it can use whole suites of symbols.

So it's a preference thing. If you guys like to get rid of it, I am not against it.